### DISCUSSION TITLE
What happen if you already have a fuseki server with a certain amount of data?

### DISCUSSION THREAD

**Discussion by ll13j1209** [afs/rdf-delta#215] [Q&A] [answered]
Hi, I already have a running fuseki server with some data, if I want to add two new servers with rdf-delta enabled, what's the recommend approach? Should I copy the data first and then setup the patch log server? Or will the patch log server automatically pick up the old data for new servers? Thank you!

**Comment by afs** [comment, marked as answer]
You can start the system of 2 Fuseki servers with some already-loaded data.

Both servers must start with copies of the same files, not two servers, separately loaded from the data.

Take a backup before starting.

Setup the Delta system, Fuseki and patch log server, with no data loaded.

Stop the Delta system Fuseki servers.

Replace the database in the servers by 

Make the two servers by stopping the original one, taking copies of the database directory and all files within it. That is, a filing system copy e.g. `rsync -avS` (the `S` is important - it preserves spare files) of the database files.

Place a directory tree in each of the desired locations for the delta installation.

Start the Delta Fuseki servers.

The delta patch server will not take a copy of the new data. It contains changes, not the original data.
